js/index.js

```diff
@@ -20,14 +20,65 @@ existingFooter.appendChild(copyright);
 const skills =["Javascript", "HTML", "CSS", "Adobe Photoshop", "gitHub"];
 
 const skillsSection = document.getElementById('skills');
-const skillsList = skillsSection.querySelector('u1');
+const skillsList = skillsSection.querySelector('ul');
 
 //For Loop
 for(let i =0; i<skills.length;i++){
     const skillItem  = document.createElement('li');
     skillItem.innerText = skills[i];
     skillsList.appendChild(skillItem);
 }
-//console.log('Skills Section:', skillsSection);
-//console.log('Skills List:', skillsList);
-// edit edit edit edit
+
+
+//message handler form submit
+const messageForm = document.forms["leave message"];
+
+messageForm.addEventListener("submit", function (event) {
+    event.preventDefault();
+    
+    //get values from the form fields
+    const usersName = event.target.usersName.value;
+    const usersEmail = event.target.usersEmail.value;
+    const usersMessage = event.target.usersMessage.value;
+
+    //console log values 
+    console.log("Name:", usersName);
+    console.log("Email:", usersEmail);
+    console.log("Message:", usersMessage);
+
+
+    //message section
+    const messageSection = document.getElementById("messages");
+    const messageList = messageSection.querySelector("ul");
+
+    //crete a new list item(li) element 
+    const newMessage = document.createElement("li");
+    newMessage.innerHTML =
+    `<a href="mailto:${usersEmail}">${usersName}</a> 
+    <span> says: ${usersMessage}</span>`
+;   
+
+    //remove button
+    const removeButton = document.createElement("button");
+    removeButton.innerText = "Remove";
+    removeButton.type = "button";
+
+    //add event listener tp the removeButton 
+    removeButton.addEventListener("click", function() {
+    const entry = removeButton.parentNode;
+    entry.parentNode.removeChild(entry);    
+  
```

Comment:
Every block should be indented.  Here this would help make it clear that the event handler for the remove button is a separate piece if code which runs at a different time.